### PR TITLE
[Nokia][Nokia-IXR7250E-SUP-10] Update BCM config for supervisor card to reduce the CPU usage

### PR DESCRIPTION
**Pull request by saksarav-nokia**
#### Why I did it
Disabled the bcmCNTR thread to reduce the CPU usage for Nokia SFM cards.
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
set bcm_start_interval=0
#### How to verify it
verified CPM card comes up with all SFMs and functioning correctly.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: device/nokia/x86_64-nokia_ixr7250e_sup-r0/Nokia-IXR7250E-SUP-10/0/config-ramon-1-0.bcm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 appl_enable_intr_init.BCM8879X=0
 appl_param_module_id.BCM8879X=1
 bcm_linkscan_interval.BCM8879X=0
-bcm_stat_interval.BCM8879X=4000000
+bcm_stat_interval.BCM8879X=0
```
Comment (judyjoseph) [inline device/nokia/x86_64-nokia_ixr7250e_sup-r0/Nokia-IXR7250E-SUP-10/0/config-ramon-1-0.bcm:5]: Making this to zero, will this affect the fabric counter poll ?

Comment (saksarav-nokia) [reply device/nokia/x86_64-nokia_ixr7250e_sup-r0/Nokia-IXR7250E-SUP-10/0/config-ramon-1-0.bcm:5]: No. It will pull the counters from HW on demand

**Review by gechiang — APPROVED** [review]

**Review by judyjoseph — APPROVED** [review]

**Comment by saksarav-nokia** [comment]
@lguohan , could you please review this PR?

**Comment by gechiang** [comment]
@StormLiangMS , @yxieca , please help review/approve for 202205 and 202305.
MSFT ADO: 25892413

**Pull request merged**